http_handler.go

```diff
@@ -549,7 +549,7 @@ func newRouter(handler *Handler) http.Handler {
 		router.HandleFunc("/sql", handler.chkAuthZ(handler.handlePostSQL, authz.Admin)).Methods("POST").Name("PostSQL")
 	}
 	// internal endpoint
-	router.HandleFunc("/sql", handler.chkAuthZ(handler.handlePostSQLPlanOperator, authz.Admin)).Headers("X-FeatureBase-Plan-Operator", "").Methods("POST").Name("PostSQLPlanOperator")
```

Comment:
Alternatively, you could have used `.HeadersRegexp()` and matched on any value, rather than on an empty value. But I guess if you're not using that value for anything it doesn't matter.
I don't have a strong opinion on whether adding a new endpoint is better.